pyuvdata/uvcal/uvcal.py

```diff
@@ -1264,6 +1682,374 @@ def _check_freq_spacing(self, *, raise_errors=True):
+            default is zero. Expected to be a float for sidereal and driftscan phase
+            centers, and an ndarray of floats of shape (Npts,) for ephem phase centers.
+        cat_lat : float or ndarray
+            Value of the latitudinal coordinate (e.g., Dec, El, b) in radians of the
+            phase center. No default unless `cat_type="unprojected"`, in which case the
+            default is pi/2. Expected to be a float for sidereal and driftscan phase
+            centers, and an ndarray of floats of shape (Npts,) for ephem phase centers.
+        cat_frame : str
+            Coordinate frame that cat_lon and cat_lat are given in. Only used
+            for sidereal and ephem targets. Can be any of the several supported frames
+            in astropy (a limited list: fk4, fk5, icrs, gcrs, cirs, galactic).
+        cat_epoch : str or float
+            Epoch of the coordinates, only used when cat_frame = fk4 or fk5. Given
+            in units of fractional years, either as a float or as a string with the
+            epoch abbreviation (e.g, Julian epoch 2000.0 would be J2000.0).
+        cat_times : ndarray of floats
+            Only used when `cat_type="ephem"`. Describes the time for which the values
+            of `cat_lon` and `cat_lat` are calculated, in units of JD. Shape is (Npts,).
+        cat_pm_ra : float
+            Proper motion in RA, in units of mas/year. Only used for sidereal phase
+            centers.
+        cat_pm_dec : float
+            Proper motion in Dec, in units of mas/year. Only used for sidereal phase
+            centers.
+        cat_dist : float or ndarray of float
+            Distance of the source, in units of pc. Only used for sidereal and ephem
+            phase centers. Expected to be a float for sidereal and driftscan phase
+            centers, and an ndarray of floats of shape (Npts,) for ephem phase centers.
+        cat_vrad : float or ndarray of float
+            Radial velocity of the source, in units of km/s. Only used for sidereal and
+            ephem phase centers. Expected to be a float for sidereal and driftscan phase
+            centers, and an ndarray of floats of shape (Npts,) for ephem phase centers.
+        info_source : str
+            Optional string describing the source of the information provided. Used
+            primarily in UVData to denote when an ephemeris has been supplied by the
+            JPL-Horizons system, user-supplied, or read in by one of the various file
+            interpreters. Default is 'user'.
+        force_update : bool
+            Normally, `_add_phase_center` will throw an error if there already exists a
+            phase_center with the given cat_id. However, if one sets
+            `force_update=True`, the method will overwrite the existing entry in
+            `phase_center_catalog` with the parameters supplied. Note that doing this
+            will _not_ update other attributes of the `UVData` object. Default is False.
```

Comment:
UVData -> UVCal

Reply:
Fixed in 766e32646768181b70ddd8d2e576d90ca57c254c.